plugin.xml

```diff
@@ -150,6 +150,11 @@
          <framework src="CoreGraphics.framework" />
          <framework src="AVFoundation.framework" />
 
+         <preference name="CAMERA_USAGE_DESCRIPTION" default=" " />
+         <config-file target="*-Info.plist" parent="NSCameraUsageDescription">
+             <string>$CAMERA_USAGE_DESCRIPTION</string>
+         </config-file>
+
          <config-file target="*-Info.plist" parent="NSLocationWhenInUseUsageDescription">
```

Comment:
@pke, I believe this is for `CameraUsesGeolocation` [preference](https://github.com/apache/cordova-plugin-camera#preferences-ios)